### DISCUSSION TITLE
Project Beta: Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by wesleyhowery** [community/community#14452] [Projects and Issues]
It would be great to have a keyboard shortcut where I can move issues up and down in the list view!

**Comment by blairio** [comment]
I totally agree. It is becoming very tedious to maintain a prioritized list by moving things with the mouse/trackpad. The classic projects had keyboard affordances to move things up and down and jump them from to the top or bottom. Can we get the same thing for boards and lists?

**Comment by pyrello** [comment]
Related: https://github.com/orgs/community/discussions/6610 and https://github.com/community/community/discussions/5890
